src/audio/OpenALAudioEngine.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "AudioEngine.h"
+
+#include <string>
+
+typedef struct ALCdevice_struct ALCdevice;
```

Comment:
`using`?

Reply:
It's going to interfere with `#include <al/al.h>` when it's included in OpenALAudioEngine